ckks/bootstrapping/simple_bootstrapper.go

```diff
@@ -0,0 +1,47 @@
+package bootstrapping
+
+import (
+	"github.com/tuneinsight/lattigo/v4/ckks"
+	"github.com/tuneinsight/lattigo/v4/rlwe"
+)
+
+// SimpleBootstrapper is an implementation of the rlwe.Bootstrapping interface that
+// uses the secret-key to decrypt and re-encrypt the bootstrapped ciphertext.
+type SimpleBootstrapper struct {
+	ckks.Parameters
+	ckks.Encoder
+	rlwe.Decryptor
+	rlwe.Encryptor
+	Counter int // records the number of bootstrapping
+}
+
+func NewSimpleBootstrapper(params ckks.Parameters, sk *rlwe.SecretKey) rlwe.Bootstrapper {
```

Comment:
This is not much more explicit than `Dummy`. What about `SkBootstrapper` or `SecretKeyBootstrapper` ?

Reply:
done